x/outline-connectivity-app/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Perhaps we should use Google's standard tsconfig: https://github.com/google/gts/blob/main/tsconfig-google.json

It's fine if we tweak it. Maybe do an import as in https://github.com/google/gts/blob/main/tsconfig.json